tests/integ/scala/test_file_operation_suite.py

```diff
@@ -460,98 +460,135 @@ def test_put_stream_negative(session, temp_stage, path1, local_testing_mode):
         assert ex_info.value.error_code == "1408" if not local_testing_mode else True
 
 
+@pytest.mark.localtest
 @pytest.mark.parametrize("with_file_prefix", [True, False])
 def test_get_one_file(
-    session, temp_stage, temp_target_directory, path1, with_file_prefix
+    session,
+    temp_stage,
+    temp_target_directory,
+    path1,
+    with_file_prefix,
+    local_testing_mode,
 ):
     stage_prefix = f"prefix_{random_alphanumeric_name()}"
     stage_with_prefix = f"@{temp_stage}/{stage_prefix}/"
 
     session.file.put(
-        f"{'file://' if with_file_prefix else ''}{path1}", stage_with_prefix
+        f"{'file://' if with_file_prefix else ''}{path1}",
+        stage_with_prefix,
+        auto_compress=not local_testing_mode,
     )
+    stage_file_name = f"{stage_with_prefix}{os.path.basename(path1)}{'.gz' if not local_testing_mode else ''}"
+    file_name = f"{os.path.basename(path1)}{'.gz' if not local_testing_mode else ''}"
     results = session.file.get(
-        f"{stage_with_prefix}{os.path.basename(path1)}.gz", str(temp_target_directory)
+        stage_file_name, str(temp_target_directory)
     )  # temp_target_directory
     results_with_statement_params = session.file.get(
-        f"{stage_with_prefix}{os.path.basename(path1)}.gz",
+        stage_file_name,
         str(temp_target_directory),
         statement_params={"SF_PARTNER": "FAKE_PARTNER"},
     )  # temp_target_directory
     try:
         assert len(results) == len(results_with_statement_params) == 1
-        assert (
-            results[0].file
-            == results_with_statement_params[0].file
-            == f"{os.path.basename(path1)}.gz"
-        )
-        assert results[0].size in (54, 55) and results_with_statement_params[
+        assert results[0].file == results_with_statement_params[0].file == file_name
+        # 10 is for local testing, 54, 55 is for non-local testing
+        assert results[0].size in (10, 54, 55) and results_with_statement_params[
```

Comment:
Let's use `if local_testing_mode` pattern, I think it's always better to be explicit.

Reply:
thanks for catching this, updated